static/styles/zulip.css

```diff
@@ -1470,8 +1470,7 @@ td.pointer {
 
     .messagebox-content {
         box-shadow: inset 0 0 0 2px hsl(215, 47%, 50%),
-            -1px -1px 0 0 hsl(215, 47%, 50%), 1px 1px 0 0 hsl(215, 47%, 50%),
-            -1px 1px 0 0 hsl(215, 47%, 50%), 1px -1px 0 0 hsl(215, 47%, 50%);
+            0 0 0 1px hsl(215, 47%, 50%);
```

Comment:
Can you write a commit message for this change? I can't tell what the story is that this is supposed to have no funcitonal changes, or if it's moving the placement slightly, or what.

Reply:
Done.